src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -428,4 +453,8 @@ public void zeroHeading() {
     m_headingOffsetRadians = getGyroAngle().in(Units.Radians);
     m_gyro.reset();
   }
+
+  public SendableChooser<Command> getAutoChooser() {
```

Comment:
Do we really need to put this in here?

Reply:
I'll move back to robot container. It was initially moved to drivetrain to set an initial pose for the auto but that's done internally within path planner.